src/Microsoft.ML.Parquet/PartitionedPathParser.cs

```diff
@@ -276,8 +280,7 @@ public void Save(ModelSaveContext ctx)
                 ctx.SaveString(sb.ToString());
             };
         }
-
-        public IEnumerable<PartitionedFileLoader.Column> ParseColumns(string path)
+        IEnumerable<PartitionedFileLoader.Column> IPartitionedPathParser.ParseColumns(string path)
```

Comment:
>IEnumerable<PartitionedFileLoader.Column> IPartitionedPathParser.ParseColumns(string path) [](start = 8, length = 90)

Why this change if the outer class is internal? Also you didn't change `ParseColumns` above, which makes me thing it does not need to be changed.

Reply:
I made this change first, and only after that made whole class internal. Thanks for the catch!

---
In reply to: [252397204](https://github.com/dotnet/machinelearning/pull/2309#discussion_r252397204) [](ancestors = 252397204)